README.md

```diff
@@ -60,14 +52,101 @@ Spinal cord MRI data:
 - T2star tra
 Brain MRI data:
 - DWI (A-P and P-A phase encoding)
-- MPM (multi-parameter mapping)
+- T1w
 
 ## naming convention
 
 sub-<site><pathology>XXX
 
 example:
-sub-zurichDCM001
+sub-montrealDCM001
+
+<ADDITIONAL NOTES IF AVAILABLE>Note: the label `bp-cspine` is used to differentiate spine images from brain.
+
+## derivatives
+
+The derivatives will be organized according to the following:
+
+https://github.com/ivadomed/ivadomed/wiki/repositories#derivatives
+
+Convention for derivatives JSON metadata:
+
+{
+  "Author": "Firstname Lastname",
+  "Date": "YYYY-MM-DD HH:MM:SS"
```

Comment:
maybe mention that date is optional (we usually include it when running the manual correction via python scripts, like what Sandrine did)

Reply:
Thanks! Added: https://github.com/neuropoly/data-management/pull/189/commits/e49fc9682379c3865c9bb3a567bb5d25c8f76036